### ISSUE TITLE
Make the extension available for the VS Live Share web client

### ISSUE THREAD

**Issue by heksesang** [scala/vscode-scala-syntax#199]
Currently it's not possible to get Scala syntax highlighting in the web client of Live Share sessions, because this extension is not declared as a web extension.

It would be nice to fix that. :)

**Comment by nicolasstucki** [comment]
@heksesang do you know the location of the documentation of how to declare as a _web extension_?

**Comment by nicolasstucki** [comment]
My guess is that we need to change the manifest but there is nothing that looks like this in https://code.visualstudio.com/api/references/extension-manifest

**Comment by nicolasstucki** [comment]
```
[2021-04-14 11:32:47.885] [window] [error] Error: Cannot install 'Scala Syntax (official)' because this extension is not a web extension.
    at E.installFromGallery (https://vscodeweb.azureedge.net/stable/3c4e3df9e89829dce27b7b5c24508306b151f30d/out/vs/workbench/workbench.web.api.js?v=2:1531:39644)
    at async Promise.all (index 0)
    at async Object.G [as settled] (https://vscodeweb.azureedge.net/stable/3c4e3df9e89829dce27b7b5c24508306b151f30d/out/vs/workbench/workbench.web.api.js?v=2:68:20994)
[2021-04-14 11:32:47.917] [window] [error] Cannot install 'Scala Syntax (official)' because this extension is not a web extension.: Error: Cannot install 'Scala Syntax (official)' because this extension is not a web extension.
    at E.installFromGallery (https://vscodeweb.azureedge.net/stable/3c4e3df9e89829dce27b7b5c24508306b151f30d/out/vs/workbench/workbench.web.api.js?v=2:1531:39644)
    at async Promise.all (index 0)
    at async Object.G [as settled] (https://vscodeweb.azureedge.net/stable/3c4e3df9e89829dce27b7b5c24508306b151f30d/out/vs/workbench/workbench.web.api.js?v=2:68:20994)
```

**Comment by nicolasstucki** [comment]
The fix in #200 did not work. That solution used an undocumented feature (see https://github.com/microsoft/vscode/issues/111043).

**Comment by heksesang** [comment]
There is now added some documentation for Web Extensions authoring here: https://code.visualstudio.com/api/extension-guides/web-extensions
